tests/test_conditions.py

```diff
@@ -181,7 +186,7 @@ def test_assert_time_exceeds(self):
         assert r.args[0].startswith("exception: (<Err.ASSERT_TIME_EXCEEDS_FAILED")
 
         # wait a second, should succeed
-        time.sleep(1)
+        x = run(remote.skip_milliseconds(ms=int_to_bytes(1000)))
```

Comment:
Same thing here, if you use `uint64` then you don't need a conversion... `ms=1000` will work as `ms` will have working `bytes` implementation.

Reply:
updated